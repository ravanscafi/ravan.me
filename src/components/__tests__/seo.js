import React from "react"
import renderer from "react-test-renderer"
import { useStaticQuery } from "gatsby"

import SEO from "../seo"

beforeEach(() => {
  useStaticQuery.mockImplementationOnce(() => ({
    site: {
      siteMetadata: {
        title: "Ravan Scafi",
        description: "Ravan Scafi's personal website.",
        author: "Ravan Scafi",
        siteUrl: "https://ravan.me",
      },
    },
    image: {
      childImageSharp: {
        fixed: {
          src: "/static/a379ede7190febde66f2453a8c9ac9c1/1f491/icon.png",
        },
      },
    },
  }))
})

describe("SEO", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<SEO title="My Title" />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
