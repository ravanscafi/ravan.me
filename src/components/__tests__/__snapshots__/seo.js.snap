// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Seo renders correctly 1`] = `null`;
